doc/user/content/serve-results/bi-tools/_index.md

```diff
@@ -0,0 +1,14 @@
+---
+title: "Use BI/data collaboration tools"
+description: "Querying results from Materialize using external BI/data collaboration tools"
+disable_toc: true
+menu:
+  main:
+    parent: serve-results
+    identifier: 'bi-tools'
+    weight: 10
+---
+
+You can integrate Materialize with various external BI/data collaboration tools.
```

Comment:
Completely unwordsmithed but maybe just a sentence that is like "We are Postgres compatible and here are some things we have seen work in practice."

Reply:
Ah, thank you!  Not getting ahead at all.  Will add. ❤️

Reply:
Heh ... reworked the "Materialize uses the PostgreSQL wire protocol, which allows it to integrate out-of-the-box with many SQL clients and other tools in the data ecosystem that support PostgreSQL" from our what is materialize page and popped it in the patch.